test/ILogger.NetStandard.Tests/ILoggerIntegrationTests.cs

```diff
@@ -0,0 +1,277 @@
+﻿// <copyright file="ILoggerIntegrationTests.cs" company="Microsoft">
+// Copyright © Microsoft. All Rights Reserved.
+// </copyright>
+
+namespace Microsoft.ApplicationInsights
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Extensions.Logging.ApplicationInsights;
+    using Microsoft.Extensions.Options;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    /// <summary>
+    /// Tests the <see cref="ILogger"/> integration for Application Insights.
+    /// </summary>
+    [TestClass]
+    public class ILoggerIntegrationTests
+    {
+        /// <summary>
+        /// Ensures that <see cref="ApplicationInsightsLogger"/> is invoked when user logs using <see cref="ILogger"/>.
+        /// </summary>
+        [TestMethod]
+        [TestCategory("ILogger")]
+        public void ApplicationInsightsLoggerIsInvokedWhenUsingILogger()
+        {
+            List<ITelemetry> itemsReceived = new List<ITelemetry>();
+
+            IServiceProvider serviceProvider = ILoggerIntegrationTests.SetupApplicationInsightsLoggerIntegration((telemetryItem, telemetryProcessor) =>
+            {
+                itemsReceived.Add(telemetryItem);
+            });
+
+            ILogger<ILoggerIntegrationTests> testLogger = serviceProvider.GetRequiredService<ILogger<ILoggerIntegrationTests>>();
+
+            testLogger.LogInformation("Testing");
+            testLogger.LogError(new Exception("TestException"), "Exception");
+            testLogger.LogInformation(new EventId(100, "TestEvent"), "TestingEvent");
+            testLogger.LogCritical("Critical");
+            testLogger.LogTrace("Trace");
+            testLogger.LogWarning("Warning");
+            testLogger.LogDebug("Debug");
+
+            Assert.AreEqual(7, itemsReceived.Count);
```

Comment:
Shouldn't it assert the content and levels?

Reply:
Done.